### DISCUSSION TITLE
How to skip extra nodes that are getting added in plot

### DISCUSSION THREAD

**Discussion by v-ngangarapu** [networkx/networkx#7107] [Q&A] [answered]
Hi
I am trying to plot by converting pytorch tensor object to networkx. After plotting I see isolated node getting added on top left corner  that doesn't exist in data I am providing as input. how I can skip the isolated node. Can you please help me with this,

here is the example code, I was trying with

import torch
import networkx as nx
import matplotlib.pyplot as plt
from torch_geometric.data import Data
from torch_geometric import utils

# Define the edge data and node data
edge_data = torch.tensor([[2, 3], [3, 2]], dtype=torch.long)
node_data = torch.tensor([[0.1947, -0.2013]], dtype=torch.float)

# Create a PyTorch Geometric Data object
data = Data(x=node_data, edge_index=edge_data)
G = utils.to_networkx(data, to_undirected=False)

# Create the plot
plt.figure(figsize=(8, 6))
nx.draw(G, with_labels=True)
plt.title("Graph")
plt.show()

The output looks as below

![image](https://github.com/networkx/networkx/assets/82965480/73e10892-9286-43d7-a675-53dad02c7ac3)

**Comment by MridulS** [comment, marked as answer]
You can remove the isolated nodes from the graph with `G.remove_nodes_from(list(nx.isolates(G)))`.
